### PR TITLE
Grammar & Additions

### PR DESCRIPTION
**Pull request by theconnectooor**
Changing "Icon" -> "ICON" & adding Rhizome community tracker, ICONgraphs Twitter, Bridge wallet and Orbit Bridge

### CODE REVIEW

**Comment by bwhli** [comment]
Haha! I just made a pull request for Icon -> ICON. Didn't see this one beforehand.

https://github.com/icon-community/awesome-icon/pull/4

**Comment by theconnectooor** [comment]
I added your Tracker as well @bwhli 😊

File: README.md [hunk 1]
```diff
@@ -1,162 +1,166 @@
...
-
-- [btp](https://github.com/icon-project/btp) - Blockchain Transmission Protocol, which is the Icon Foundation's core interoperability product
+_Tools and software related to blockchain interoperability in the ICON ecosystem_
+ 
+- [btp](https://github.com/icon-project/btp) - Blockchain Transmission Protocol, which is the ICON Foundation's core interoperability product
 - [btp-litepaper](https://github.com/icon-project/btp-litepaper) - Lite explainer paper for BTP. [Also available here](https://icon.community/assets/btp-litepaper.pdf)
-- [Nexus](https://github.com/icon-project/Nexus) - Interoperability frontend for using the Blockchain Transmission Protocol with the Icon main network as a routing hub
-
+- [Nexus](https://github.com/icon-project/Nexus) - Interoperability frontend for using the Blockchain Transmission Protocol with the ICON main network as a routing hub
+- [Orbit Bridge](https://bridge.orbitchain.io/) - Bridging option available on for the ICON blockchain ecosystem
```
Comment (han-so1omon) [inline README.md]: Please link to Github or Docs. If neither are available, then link to homepage

File: README.md [hunk 2]
```diff
@@ -1,162 +1,166 @@
...
-_Tools and software related to tracking the state of the Icon blockchain_
-
-- [Icon Community Tracker](https://tracker.icon.community/) - A blockchain tracker for the Icon main and test networks that can be [run locally as well](https://github.com/sudoblockio/icon-tracker)
-- [Icon Watch](https://iconwat.ch/) - A blockchain tracker made by [Blockmove](https://blockmove.eu/)
-
+_Tools and software related to tracking the state of the ICON blockchain_
+ 
+- [ICON Community Tracker](https://tracker.icon.community/) - A blockchain tracker for the ICON main and test networks that can be [run locally as well](https://github.com/sudoblockio/icon-tracker)
+- [ICON Watch](https://iconwat.ch/) - A blockchain tracker made by [Blockmove](https://blockmove.eu/)
+- [Rhizome Community Tracker](https://alpha.tracker.rhizome.dev/governance/) - A blockchain explorer built by the team members from Rhizome.
```
Comment (han-so1omon) [inline README.md]: Please link to base page, not governance page

File: README.md [hunk 3]
```diff
@@ -1,162 +1,166 @@
...
-- [Hana wallet](https://chrome.google.com/webstore/detail/hana/jfdlamikmbghhapbgfoogdffldioobgl) - Chrome extension wallet for cryptocurrencies in the Icon ecosystem
-- [IconEx wallet](https://github.com/icon-project/iconex_chrome_extension) -  ICONex Chrome extension wallet for cryptocurrencies in the Icon ecosystem
-- [MyIconWallet](https://github.com/Hana-Technology/myiconwallet-react) - Desktop and mobile wallet for cryptocurrencies in the Icon ecosystem 
-
+_Cryptocurrency wallets built with the ICON ecosystem in mind_
+ 
+- [Hana wallet](https://chrome.google.com/webstore/detail/hana/jfdlamikmbghhapbgfoogdffldioobgl) - Chrome extension wallet for cryptocurrencies in the ICON ecosystem
+- [IconEx wallet](https://github.com/icon-project/iconex_chrome_extension) -  ICONex Chrome extension wallet for cryptocurrencies in the ICON ecosystem
+- [MyIconWallet](https://github.com/Hana-Technology/myiconwallet-react) - Desktop and mobile wallet for cryptocurrencies in the ICON ecosystem
+- [Bridge wallet](https://bridgepay.money/) - A fiat onramp for USD that is integrated for use with the ICON ecosystem
```
Comment (han-so1omon) [inline README.md]: Please link to Github or Docs. If neither are available, then link to homepage

**Review by han-so1omon — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,162 +1,168 @@
...
-  - [Governance](#governance)
-  - [Interoperability](#interoperability)
-  - [NFT](#nft)
-  - [Oracle](#oracle)
-  - [RPC](#rpc)
-  - [Templating](#templating)
-  - [Testing](#testing)
-  - [Tracker](#tracker)
-  - [Wallet](#wallet)
+ - [Core ICON blockchain](#core-icon-blockchain)
```
Comment (han-so1omon) [inline README.md]: Keep the indentation here. Do a preview of the file to see what I mean. I want the project subsections to be indented below the Project section

**Review by han-so1omon — CHANGES_REQUESTED** [review]

**Review by han-so1omon — APPROVED** [review]

**Pull request merged**